src/__tests__/Toast.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import * as _solidIcons from '@fortawesome/free-solid-svg-icons';
+import { mount } from 'enzyme';
+import 'jest-enzyme';
+import * as React from 'react';
+
+import { IconLibrary } from '../Icon';
+import { Toast } from '../Toaster/Toast';
+
+function expectToast(type: string, iconClassName: string) {
+  const toast = mount(
+    <Toast
+      content={{
+        title: 'Title',
+        body: 'Body',
+      }}
+      type={type}
+    />
+  );
+
+  expect(toast.find(`svg.${iconClassName}`)).toHaveLength(1);
```

Comment:
I guess you want to find an icon here, right?

```toast.find(Icon).fitlerWhere(node => node.prop('icon') === iconName)```

Reply:
nice! that's exactly what I needed :)